src/keys.ts

```diff
@@ -15,8 +15,11 @@ export namespace JWTAuthenticationBindings {
   );
 }
 
-export namespace OtherServicesBindings {
-  export const HASH_PASSWORD = BindingKey.create<HashPassword>(
-    'services.hash_password',
+export namespace BcryptHasherBindings {
+  export const BCRYPT_HASHER = BindingKey.create<BcryptHasher>(
+    'services.bcrypt.hasher',
```

Comment:
In dependency inversion, the bindings are usually defined via interfaces, to allow consumers to provide any implementation matching the required contract.

I am proposing to change this binding to something like

```ts
export const PASSWORD_HASHER = BindingKey.create<PasswordHasher>(
  'services.password-hasher'
);
```